Livecheckables/bareos-client.rb

```diff
@@ -0,0 +1,4 @@
+class BareosClient
+  livecheck :url   => "https://github.com/bareos/bareos.git",
+            :regex => /^Release\/(\d+(?:\.\d+)+)$/
```

Comment:
```suggestion
            :regex => %r{^Release/(\d+(?:\.\d+)+)$}i
```

`brew style` wants this to use the `%r{}` regex literal syntax, due to the forward slash. Additionally, we may want to make this case insensitive (adding `i` after the regex), so it could also match tags starting with `release` (should they ever appear).

Reply:
👍